src/services/eventbridge.ts

```diff
@@ -0,0 +1,57 @@
+import { EventBridge } from 'aws-sdk';
+import { IEventEntry } from '../interfaces/IEventEntry';
+import { IEntries } from '../interfaces/IEntries';
+import { IDynamicsBooking } from '../interfaces/IDynamicsBooking';
+import { ISendResponse } from '../interfaces/ISendResponse';
+import logger from '../util/logger';
+import config from '../config';
+
+const eventbridge = new EventBridge();
+const sendBooking = async (
+  lineItems: IDynamicsBooking[],
+): Promise<ISendResponse> => {
+  const sendResponse: ISendResponse = {
+    SuccessCount: 0,
+    FailCount: 0,
+  };
+  
+  logger.info('sendBooking starting');
+
+  for (const lineItem of lineItems) {
+    // eslint-disable-next-line security/detect-object-injection
+    logger.info(
+      `sending test booking with test date ${lineItem.testDate} for vehicle ${lineItem.vrm} at test station ${lineItem.pNumber} to eventbridge...`,
+    );
+
+    try {
+      const entry: IEventEntry = {
+        Source: config.aws.eventBusSource,
+        // eslint-disable-next-line security/detect-object-injection
```

Comment:
do we need these eslint disable statements? I have removed them and ran lint and didn't get any errors